BGL/doc/BGL/Concepts/HalfedgeGraph.h

```diff
@@ -55,5 +55,11 @@ Expression                              | Returns
 `next(h, g)`                            | `halfedge_descriptor`                                                        | The next halfedge around its face.
 `prev(h, g)`                            | `halfedge_descriptor`                                                        | The previous halfedge around its face.
 `boost::graph_traits<G>::%null_halfedge()` | `halfedge_descriptor`                                                     | Returns a special halfedge that is not equal to any other halfedge.
+
+\cgalHeading{Invariants}
+
+`halfedge(edge(h,g))==h`
```

Comment:
Le Monday 11 July 2016 09:22:28 Guillaume Damiand a écrit :
> > +`halfedge(edge(h,g))==h`
> 
> I think like @lrineau , this invariant cannot be true for all halfedges
> because each edge is associated with only one of its halfedge.

As Sébastien pointed out, that means that any "edge" has at least two 
different "oriented" representations, that are considered as equal by 
`operator==`.

-- 
Laurent Rineau, PhD
R&D Engineer at GeometryFactory           http://www.geometryfactory.com/
Release Manager of the CGAL Project       http://www.cgal.org/